integration_tests/mockData/actionPlanByPrisonNumberData.ts

```diff
@@ -46,7 +46,7 @@ const actionPlans = {
             createdAt: '2023-07-20T09:29:15.386Z',
             updatedBy: 'auser_gen',
             updatedAt: '2023-07-20T09:29:15.386Z',
-            targetCompletionDate: '2024-02-29',
+            targetCompletionDate: '2124-01-29',
```

Comment:
Unrelated to this PR, this fixes the build on `main`. I don't `02-29` (i.e. a leap day) was relevant in any tests and I see no reason not to set this to a date in the distant future.